### ISSUE TITLE
Help: OSX update just took my cmd-shift-v

### ISSUE THREAD

**Issue by hedefalk** [naotaka/ClipMenu#35]
I've been using cmd-shift-v for clipmenu shortcut but latest update of OSX took this as "paste and match style" and I can't seem to find any way to unset that. Any tips highly appreciated!

**Comment by hedefalk** [comment]
Seems there was something broken with ClipMenu actually. Just installed Clipy instead which seems to be more alive and was working on latest OSX: https://github.com/Clipy/Clipy/

**Comment by hasnat** [comment]
seems break if you visit clipmenu preferences in high sierra
to make it work again you can remove the cmd+shift+v shortcut and add it back again from preferences
or reboot like its windows 98

**Comment by one19** [comment]
I just had to restart clipmenu to get the preference back 😄

**Comment by hasnat** [comment]
yes seems like lately if you only restart clipmenu it brings back the cmd+shift+v
